db.py

```diff
@@ -0,0 +1,35 @@
+import os
+from sqlalchemy import create_engine
+from urllib.parse import quote_plus
+
+
+import psycopg2
+
+def get_connection():
+    #establishing the connection
+    conn = psycopg2.connect(
+        database="postgres",
+        user='newuser',
```

Comment:
This can be passed as ENV variable so that during deployment, devops will be able to set the actual user, password, host and port as environment variable to the application

Reply:
done.. planned to do after dev testing..